weaver/wps_restapi/processes/processes.py

```diff
@@ -518,7 +520,9 @@ def add_local_process(request):
             raise HTTPUnprocessableEntity("Invalid parameter 'processDescription.process.owsContext.offering.content'.")
         package = None
         reference = content.get('href')
-    elif deployment_profile_name.endswith('workflow') or deployment_profile_name.endswith('application'):
+    elif deployment_profile_name:
+        if not any(deployment_profile_name.endswith(typ) for typ in [PROCESS_APPLICATION, PROCESS_WORKFLOW]):
```

Comment:
I guess that I found the answer myself... So Dockerized and WpsApplication both end with application (but endswith isn't case sensitive?)

Reply:
there's a lower a few lines above